src/HttpReply.h

```diff
@@ -132,6 +132,10 @@ class HttpReply: public Http::Message
     /// \retval 0 implies that more data is needed to parse the response prefix
     size_t parseTerminatedPrefix(const char *, size_t);
 
+    /// approximate size of a "status-line CRLF headers CRLF" sequence
+    /// \sa HttpRequest::prefixLen()
+    size_t prefixLen() const;
```

Comment:
The name of this new method -- prefixLen() -- was chosen to match the name of the existing HttpRequest::prefixLen() method. It is not the best possible name (e.g., approximatePrefixLength() would be better), but I think it is best to keep the two method names in sync. We could even make them virtual, but there is no need to add that overhead right now.

If you would like, I can followup with a PR that renames the two methods, but please do not make that renaming a precondition for _this_ PR, to avoid creating a yet another long chain of minimal PRs getting stuck in the review queue.